ts/components/ui/LightModal.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Implements a "lite" Modal using Context API
+ */
+
+import { View } from "native-base";
+import React from "react";
+import { StyleSheet } from "react-native";
+
+interface LightModalContextInterface {
+  component: React.ReactNode;
+  showModal: (component: React.ReactNode) => void;
```

Comment:
why not just `show()` and `hide()` ?

Reply:
I think showModal() and hideModal() are better names because they are methods of LightModalProvider, the Provider let you show the modal.
If we rename for example hideModal() to hide() it seem that you are "hiding" the Provider but the truth it that you are hiding the modal.